draft-ietf-quic-qpack.md

```diff
@@ -279,14 +281,14 @@ table entries which have been acknowledged, but this could mean using
 literals. Since literals make the header block larger, this can result in the
 encoder becoming blocked on congestion or flow control limits.
 
-### Largest Known Received
+### Known Received Count
 
 In order to identify which dynamic table entries can be safely used
 without a stream becoming blocked, the encoder tracks the absolute index of the
-decoder's Largest Known Received entry.
+decoder's Known Received Count entry.
```

Comment:
What is the decoder's Known Received Count entry?  That doesn't really make sense to me, maybe just this sentence needs rewording.

What the encoder is tracking is the state of the decoder.  The state of either the encoder or the decoder can be uniquely identified by the number of inserts that it has performed on its dynamic table.  

Would it be helpful to call the a table's 'State' or 'Version'?  Or maybe change it from "Known Received Count" to "Decoder's Known Insert Count"?

Reply:
I failed to update this text.  I will do that.